docs/content/integrations/backend/docker-model-runner.md

```
@@ -0,0 +1,620 @@
+| Engine | Model Format | Use Case |
+|--------|-------------|----------|
+| **llama.cpp** | GGUF (quantised) | Default; all platforms; CPU and GPU |
+| **vLLM** | safetensors | Linux + NVIDIA; high throughput |
+
+The `/engines/v1/...` paths use automatic engine selection. The explicit `/engines/llama.cpp/v1/...` and `/engines/vllm/v1/...` paths target a specific engine regardless of model format.
+
+### Model Naming Convention
+
+DMR uses a `namespace/name` format for model identifiers, matching Docker Hub image conventions:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)